bundles/org.openhab.binding.teslapowerwall/src/main/resources/OH-INF/binding/binding.xml

```diff
@@ -0,0 +1,10 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
binding.xml is no longer used as of openHAB 4.0.0 Please remove this file.

Reply:
done.